php/php.editor/src/org/netbeans/modules/php/editor/verification/UnusableTypesHintError.java

```diff
@@ -430,18 +430,15 @@ private void checkDuplicateType(List<Expression> types) {
                 }
                 if (checkedTypes.contains(name)) {
                     createDuplicateTypeError(type, typeName);
-                    return;
                 } else if (checkedTypes.contains(Type.BOOL)) {
                     // bool|false bool|true
                     if (Type.FALSE.equals(name) || Type.TRUE.equals(name)) {
                         createDuplicateTypeError(type, typeName);
-                        return;
```

Comment:
Unit tests?

Reply:
I've added more test cases covering also combinations of boolean type error with duplicate class error.